src/Login.js

```diff
@@ -90,7 +90,7 @@ const Login = (props) => {
         textColor="white"
         bgColor={darkGreen}
         btnLabel="Let's go!"
-        Press={handleLogin}
+        Press={() => props.navigation.navigate("Home")}
```

Comment:
No, when the button is pressed if should call the function handleLogin. Inside the function it should map to Home